src/Microsoft.Health.ControlPlane.Core/Features/Rbac/IRbacService.cs

```diff
@@ -19,5 +19,13 @@ public interface IRbacService
         Task<UpsertResponse<IdentityProvider>> UpsertIdentityProviderAsync(IdentityProvider identityProvider, string eTag, CancellationToken cancellationToken);
 
         Task DeleteIdentityProviderAsync(string name, string eTag, CancellationToken cancellationToken);
+
+        Task<Role> GetRoleAsync(string name, CancellationToken cancellationToken);
+
+        Task<UpsertResponse<Role>> UpsertRoleAsync(Role role, string eTag, CancellationToken cancellationToken);
+
+        Task<IEnumerable<Role>> GetRoleForAllAsync(CancellationToken cancellationToken);
```

Comment:
Lets make this name the same as in the `IControlPlaneDataStore` since the rest match up between the two interfaces.

Reply:
Sorry I wasn't clear - I meant we should name this method `GetAllRolesAsync` in both the rbacService and controlPlaneDataStore